meetings/2022-05-09.md

```diff
@@ -0,0 +1,61 @@
+# W3C Solid Community Group: Test Suite Panel
+
+* Date: 2022-05-09T14:00:00Z
+* Call: https://meet.jit.si/solid-test-suite
+* Chat: https://gitter.im/solid/test-suite
+* Repository: https://github.com/test-suite-panel
+
+
+## Present
+* [Sarven Capadisli](https://csarven.ca/#i)
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Join queue to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* name
+
+
+### Introductions
+* name: text
+
+---
+
+## Topics
+
+* SC: The Test Suite Panel is a "horizontal" panel. We need to revisit/clarify our understanding of the Test Suite Panel alongside other panels in the CG as well as contributors to specs and implementers.
+
```

Comment:
MJ: We need to decide how to report about servers that do not implement WAC (a required part of the spec).
MJ: We need to decide how to report about servers that do not support concurrency (not required, but useful).
MJ: When we become aware of a bug in a server, what (if any) should be the process for reporting this to the implementers vs in technical discussions in a small audience vs in proactive publications to a broader public.

Reply:
>We need to decide how to report about servers that do not implement WAC (a required part of the spec).

If an implementation is not implementing a specification or need not be tested, it'll have an outcome to untested or inapplicable. It is possible for a test suite to ignore that information and still carry out a test and give an outcome of failed but that particular test would likely not receive a positive assessment by specification implementers.

If there is any blurb about this in the Charter, it'd be about deliverables using existing standards, e.g., EARL for report testings. It would not be specific to any specification. Covered by PROPOSAL-1.

>We need to decide how to report about servers that do not support concurrency (not required, but useful).

Covered by PROPOSAL-3.

>When we become aware of a bug in a server, what (if any) should be the process for reporting this to the implementers vs in technical discussions in a small audience vs in proactive publications to a broader public.

This is not needed for the Charter.

I've answered this at https://gitter.im/solid/test-suite?at=627002cefa846847c9474ab8

>* Software bugs: Create an issue in relevant public repository. If not public repository, follow project maintainer's recommendations on how to report bugs.
>* Tests and results: Provide metadata and as much context as necessary to help the consumer better assess information. Refer only to authoritatively published test results and distinguish other results as unauthoritative/preliminary/unverified.